src/main/java/duke/Event.java

```diff
@@ -0,0 +1,99 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a Task with a start and end time.
+ */
+public class Event extends Task {
+
+    protected LocalDateTime from;
```

Comment:
could these be more explicitly named? e.g. fromDate or something like that.

Reply:
Thank you for your feedback! I will make changes accordingly.